package.json

```diff
@@ -148,7 +149,7 @@
     "stylelint-processor-styled-components": "=1.9.0",
     "surge": "=0.21.3",
     "svgo": "^2.2.0",
-    "typescript": "^4.3.2",
+    "typescript": "^4.4.3",
```

Comment:
Vu qu'on upgrade vers typescript 4.4, il faudrait upgrade prettier vers sa version compatible:
`"eslint-config-prettier": "^8.3.0"`
et
`"prettier": "^2.4.0",`

Reply:
@guillaumewttj we can keep this for an another PR ?

Reply:
yes, im creating one based on this branch. We will merge it when this one is ok 🙂

Reply:
All good in #920